sgv2-graphqlapi/src/main/java/io/stargate/sgv2/graphql/impl/GraphqlServiceServer.java

```diff
@@ -15,6 +15,7 @@
  */
 package io.stargate.sgv2.graphql.impl;
 
+import com.codahale.metrics.MetricRegistry;
```

Comment:
We should definitely use Micrometer and not drop-wizard stuff.. The `MeterRegistry` should be available via the `Metrics metrics` instance. Not only it's better, but it would enable easier move to the Quarkus, as there Micrometer is used.

Reply:
I am not against that and was wondering about Micrometer. The only concern I have is wrt how metrics are reported and accessible: right now `/metrics` endpoint is quite convenient. As long MM can be hooked similarly it does sound like a good idea to future-proof code.

Reply:
Never mind above. Converted `Timer`s used to Micrometer as suggested. Thanks @ivansenic!